### ISSUE TITLE
systemd service file

### ISSUE THREAD

**Issue by ojaksch** [mikekov/AtariServer#3]
Personally I don't like the use of /etc/rc.local . Maybe as a last resort, but it's ugly. So let's prefer a systemd service  file (**atariserver.service**):
```
[Unit]
Description=AtariServer - 8-bit Atari floppy emulator and web server for Raspberry Pi
ConditionPathExists=/dev/ttyAMA0

[Service]
Type=simple
ExecStart=/usr/local/bin/atariserver -w /home/pi/AtariServer-master/web-root -b /home/pi/AtariServer-master/atr/hisioboot-atarisio.atr -d /home/pi/files -C
StandardOutput=tty
RemainAfterExit=yes

[Install]
WantedBy=multi-user.target
```
It is not perfect for now as it starts **before** wpa_supplicant on my rpi0w for example, but it's a good starting point, I think.

**Comment by mikekov** [comment]
Maybe "After=network.target" in [Unit] would help?

Frankly, I just wanted to have the job done, so I was not too concerned with how atariserver starts.

**Comment by ojaksch** [comment]
> Maybe "After=network.target" in [Unit] would help?

Absolutely, yes. Or even better (accoding to [https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/](url)):

`After=network-online.target`
`Wants=network-online.target`


> Frankly, I just wanted to have the job done, so I was not too concerned with how atariserver starts.

Comprehensible. And as it's online and working now we have to cleanup and to optimise this baby :)

**Comment by mikekov** [comment]
Cool. If you have enough enthusiasm, please send me a PR.
Install script needs to check if this service entry is present and add it only if necessary.
